docker-compose.yml

```diff
@@ -150,6 +150,7 @@ services:
             - data-search:/usr/share/elasticsearch/data
         environment:
             discovery.type: single-node
+            index.blocks.read_only_allow_delete: null
```

Comment:
so weird to see environment variables with a `.`, but the format seems to already be in place for `discovery.type`